docs/Using Fleet/Understanding-host-vitals.md

```diff
@@ -462,7 +462,7 @@ SELECT
   'Browser plugin (Chrome)' AS type,
   'chrome_extensions' AS source,
   '' AS vendor,
-  path AS installed_path
+  '' AS installed_path
```

Comment:
@lucasmrod I have 3 potential solutions in this 1 PR, I was hoping to get your opinion on.

Even though `path` is set to `""` in `chrome_extensions.ts`, the UI is receiving an  error in Dogfood when running `select path from chrome_extensions` so my hunch was setting `path AS installed_path` was the issue.

1. The first one is this, setting the installed_path to an empty string for the host vitals of a chrome host because the Chrome API doesn't actually return a path.

This is the change in the docs to let the user know the chrome extension software will not have an installed path. Code changes are found in `queries.go`

Reply:
This part makes sense to me.

Double checking: This is an auto generated doc. Did you run `make generate-doc` (after modifying the `queries.go`)?

Reply:
No I didn't! Good to know!!

Reply:
Just ran it just in case, no changes so we're good to go